bootstrap-scripts/package_install.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -v
+
+set -ex
+if [ '$pnda_mirror$' != '$''pnda_mirror''$' ]; then 
+  PNDA_MIRROR=$pnda_mirror$
+fi
+DISTRO=$(cat /etc/*-release|grep ^ID\=|awk -F\= {'print $2'}|sed s/\"//g)
+if [ "x$DISTRO" == "xubuntu" ]; then
+  export DEBIAN_FRONTEND=noninteractive
+  if [ "x$PNDA_MIRROR" != "x" ]; then
```

Comment:
I don't think that's necessary, the CLI has a check that the mirror is defined and accessible and the CLI then sets up this variable, so it's OK to assume it is defined.